tests/e2e/steps_permissionless_ics.go

```diff
@@ -0,0 +1,213 @@
+package main
+
+import (
+	"time"
+
+	clienttypes "github.com/cosmos/ibc-go/v8/modules/core/02-client/types"
+	e2e "github.com/cosmos/interchain-security/v6/tests/e2e/testlib"
+)
+
+// stepsPermissionlessICS tests
+// - starting multiple permissionless consumer chains with the same chain ID
+// - opt-in of a validator on two different chains with the same chain ID
+// - taking ownership of a consumer chain
+func stepsPermissionlessICS() []Step {
+	s := concatSteps(
+		[]Step{
+			// Start the provider chain
+			{
+				Action: StartChainAction{
+					Chain: ChainID("provi"),
+					Validators: []StartChainValidator{
+						{Id: ValidatorID("alice"), Stake: 100000000, Allocation: 10000000000},
+						{Id: ValidatorID("bob"), Stake: 200000000, Allocation: 10000000000},
+						{Id: ValidatorID("carol"), Stake: 300000000, Allocation: 10000000000},
+					},
+				},
+				State: State{
+					ChainID("provi"): ChainState{
+						ValPowers: &map[ValidatorID]uint{
+							ValidatorID("alice"): 100,
+							ValidatorID("bob"):   200,
+							ValidatorID("carol"): 300,
+						},
+					},
+				},
+			},
+
+			// Initialize a permissionless chain with ChainID `consu`
+			// - create the consumer chain
+			// - opt-in a validator
+			// - launch the chain
+			// - TODO (PERMISSIONLESS): 'start the chain'. Note: due to hermes we can't yet start two chains with the same ChainID as it's not yet supported
```

Comment:
Why do we need to start the second chain? I think from testing perspective, it's okay if only one is up and running (?)
Is this a general Hermes issue (cannot connect 2 chains with the same `chainId`)?

Reply:
yes, confirmed with hermes team that the identifier used in the hermes config is also used to double check that this matches the chain-id.
From an e2e testing perspective it's better IMO to have the 2nd chain also started to verify that there are no side effects, especially with issues reported by hermes. As this can't be tested due to hermes adoption to permissionless ICS the current setup is the best we can get at the moment.

Reply:
I'm not sure starting the second chain is relevant.

Any side effects can be confirmed in unit and integration tests.

Per IBC rules it is fine to have 2 chains with the same chain_id. Hermes may have issues connecting 2 chains with the same chain_id, but that's not relevant for ICS.